src/plugins/intel_gpu/src/kernel_selector/cl_kernels/gemm_tiled_opt.cl

```diff
@@ -281,17 +316,43 @@ KERNEL(gemm_tiled_opt)(
         }
     } // Loading leftovers of the matrix A and tile C calculation end
 #endif // TILE_K_NOT_DIVISIBLE
+#endif // IS_DYNAMIC
 
 #if HAS_FUSED_OPS && FUSED_OPS_CAN_USE_PRELOAD
+#if IS_DYNAMIC
+    FUSED_OPS_PRELOAD_SCALAR;
+#else // IS_DYNAMIC
 #if TILE_N_NOT_DIVISIBLE || B_VEC_SIZE == 1
```

Comment:
|| IS_DYNAMIC

Reply:
For the same reason as above, this code was separated.